### PR TITLE
feat(insights): update docs for resource explorer v2

### PR DESCRIPTION
**Pull request by tyler-dunkel**
### Proposed changes

These changes update the docs for the resources page to reflect the changes we are releasing with the resources page v2

### Related issues (optional)

[Resource explorer epic](https://github.com/pulumi/pulumi-service/issues/21343)

### CODE REVIEW

**Review by lblackstone — APPROVED** [review]

File: content/docs/pulumi-cloud/insights/_index.md [hunk 1]
```diff
@@ -22,16 +22,18 @@ Pulumi Insights provides advanced search, analytics, and AI for your infrastruct
...
+- **Resource Search Favoriting and Sharing**: Save your updated view as a favorite in the left nav like stacks or share it with your teammates.
+
+- **Resource Search AI Assist**: (experimental) Use natural language processing to help craft search queries to explore your data.
 
 - **Data Export**: Export your Pulumi resources into your business intelligence tool of choice to go even further.
 
-- **Resource Search AI Assist**: (experimental) Use natural language processing to help craft search queries to explore your data.
+- **Cloud Import**: Bring your own cloud provider account and import all your existing resources into Pulumi to see how things break down. Zero code required.
 
 ## Resource Search
```
Comment (interurban) [inline content/docs/pulumi-cloud/insights/_index.md:38]:
```suggestion
## Resource search
```

File: content/docs/pulumi-cloud/insights/_index.md [hunk 2]
```diff
@@ -41,43 +43,57 @@ Resource Search can be accessed directly from the Pulumi Cloud dashboard or from
...
 
-The columns displayed on results can be modified to show or hide information by selecting the gear icon.
+![Resource Search Options Gear](gear-icon-dropdown.png)
 
-![Resource Search Columns](search-columns.png)
+![Resource Search Column Chooser](column-chooser.png)
 
 Selecting a column header will modify the query to sort by that column.
 
 ### Resource Search Aggregates
```
Comment (interurban) [inline content/docs/pulumi-cloud/insights/_index.md:70]:
```suggestion
### Resource search aggregates
```

File: content/docs/pulumi-cloud/insights/_index.md [hunk 3]
```diff
@@ -41,43 +43,57 @@ Resource Search can be accessed directly from the Pulumi Cloud dashboard or from
...
+
+The **Row Groups Header** allows you to drag columns from the table to enable grouping by that column. You can group by multiple columns at once. In this example, we have grouped by stack and modified so that we can see a breakdown of resources by stack and when they were last modified.
+
+![Resource Search Grouping](resource-search-grouping.png)
+
+Selecting the **"X" in the search bar** will remove all previously selected filters.
+
+![Resource Search Clear Filters](clear-search.png)
+
+### Resource Search Favoriting
```
Comment (interurban) [inline content/docs/pulumi-cloud/insights/_index.md:88]:
```suggestion
### Resource search favoriting
```

File: content/docs/pulumi-cloud/insights/_index.md [hunk 4]
```diff
@@ -41,43 +43,57 @@ Resource Search can be accessed directly from the Pulumi Cloud dashboard or from
...
 ### Resource Search Aggregates
 
-The **Advanced filtering** menu can be expanded to apply additional filters to your query and to view finer-grained resource counts.
+The **Column Filter** menu can be expanded to apply additional filters to each column and to view finer-grained resource sets.
+
+![Resource Search Column Filter](column-filter.png)
+
+In this example, the query has been restricted to the "dev" stack.
+
+### Resource Search Grouping
```
Comment (interurban) [inline content/docs/pulumi-cloud/insights/_index.md:78]:
```suggestion
### Resource search grouping
```

File: content/docs/pulumi-cloud/insights/_index.md [hunk 5]
```diff
@@ -41,43 +43,57 @@ Resource Search can be accessed directly from the Pulumi Cloud dashboard or from
...
-In this example, the query has been restricted to the "my-stack" stack.
+![Resource Search Favorites left nav](favorites-left-nav.png)
 
-The counts next to each value show that this stack has 18 subnets, and 366 AWS resources in total.
+All your saved favorites can be seen as quick links on the **left nav**.
 
-Selecting **Clear filters** will remove all previously selected filters.
+![Resource Search Favorites button](favorites-button.png)
 
 ## Data Export
```
Comment (interurban) [inline content/docs/pulumi-cloud/insights/_index.md:98]:
```suggestion
## Data export
```

File: content/docs/pulumi-cloud/insights/search.md [hunk 6]
```diff
@@ -210,6 +210,16 @@ Examples:
...
+
+The category the resource belongs to.
+
+Examples:
+
+- compute
+- storage
+- data
+
 ### Exact Matching
```
Comment (interurban) [inline content/docs/pulumi-cloud/insights/search.md:223]:
```suggestion
### Exact matching
```

File: content/docs/pulumi-cloud/insights/search.md [hunk 7]
```diff
@@ -317,18 +327,17 @@ If we want to refine this to resources with _some_ tags but no `"stack"` tag spe
 
 Existence queries do not return resources where the property is an empty object (`{}`), array (`[]`), or `null`. A resource with an output of `{"tags": {}}` would not be captured by an existence query for `.tags:`.
 
-## Advanced Filtering
+## Column Filtering
```
Comment (interurban) [inline content/docs/pulumi-cloud/insights/search.md:330]:
```suggestion
## Column fltering
```

File: content/docs/pulumi-cloud/insights/_index.md [hunk 8]
```diff
@@ -41,43 +43,57 @@ Resource Search can be accessed directly from the Pulumi Cloud dashboard or from
 
 By default, you will see a table with the resources you have access to, ordered by most recently updated.
 
-A count is shown the upper-right corner with the total number of resources matched by this query -- in this case, we have 70 resources.
-
-![Resource Search Table](search-table.png)
+A count is shown the upper-right corner with the total number of resources matched by this query -- in this case, we have 24 resources.
```
Comment (interurban) [inline content/docs/pulumi-cloud/insights/_index.md:46]:
```suggestion
A count is shown in the upper-right corner with the total number of resources matched by this query -- in this case, we have 24 resources.
```

File: content/docs/pulumi-cloud/insights/search-table-v2.png
Comment (interurban) [inline content/docs/pulumi-cloud/insights/search-table-v2.png:1]: it's hard to read the text on this image; maybe shrink your browser window for the screenshot?

File: content/docs/pulumi-cloud/insights/search.md [hunk 9]
```diff
@@ -317,18 +327,17 @@ If we want to refine this to resources with _some_ tags but no `"stack"` tag spe
...
 
-Expanding the "Advanced filtering" menu shows your results broken down by type, package, stack, and project.
-The values shown in each column and the top values for that particular dimension, along with a count of how many resources share that value.
+Expanding the "Column Filter" menu shows you the set of values found for that column along with a count for each value.
+You can then select a specific value or collection of values to filter your result set.
 
+TODO: change image
```
Comment (interurban) [inline content/docs/pulumi-cloud/insights/search.md:335]: you've got a todo here

**Review by interurban — APPROVED** [review]
Can you take a look at the couple comments, approving so you can merge when ready.

**Pull request merged**